Python/Product/Common/Infrastructure/StringExtensions.cs

```diff
@@ -97,5 +97,7 @@ public static class StringExtensions {
             }
             return str;
         }
+
+        public static string AddQuotes(this string str) => "\"" + str + "\"";
```

Comment:
`$"\"{str}\""` - does it really need an extension method?

Reply:
I can remove this.

---
In reply to: [153326133](https://github.com/Microsoft/PTVS/pull/3395#discussion_r153326133) [](ancestors = 153326133)